README.md

```diff
@@ -1,5 +1,31 @@
 # SMLHelper
-A complete library that helps you out on your Subnautica modding adventures by making adding new items, models, sprites a LOT easier. Check out the wiki page.
+SMLHelper is a complete library that helps you out on your Subnautica modding adventures by making adding new items, changing items, adding models, sprites a LOT easier. Check out the wiki page for details on how to use it.
 
-# Contributing
+## Contributing
 We would love to have people contribute to SMLHelper. To get started, first fork the repo and then clone it, and then copy Assembly-CSharp.dll and Assembly-CSharp-firstpass.dll (located in your Subnautica_Data/Managed folder) to the Dependencies folder. Then, load up the solution, make your edits and PR them!
+
+## Alphabetical list of mods requiring SMLHelper
```

Comment:
Added an alphabetical list of mods requiring smlhelper. I will update this myself everytime a new mod releases